docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1386,8 +1386,9 @@ Query Parameters:
 
 -   **dockerfile** - Path within the build context to the Dockerfile. This is
         ignored if `remote` is specified and points to an individual filename.
--   **t** – A repository name (and optionally a tag) to apply to
-        the resulting image in case of success.
+-   **t** – A list of repository names to apply to the resulting image in case
```

Comment:
omit "in case of success" document as if success were a given. so

-->

A name and optional tag to apply to the image in the `name:tag` format. If you omit the `tag` the default `latest` value is assumed. You can provide one or more **t** parameters.